src/vm-repair/azext_vm_repair/custom.py

```diff
@@ -0,0 +1,293 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import subprocess
+import json
+from datetime import datetime
+from uuid import uuid4
+
+from knack.log import get_logger
+
+from azure.cli.command_modules.vm.custom import get_vm, _is_linux_os
+from azure.cli.command_modules.storage.storage_url_helpers import StorageResourceIdentifier
+from azure.cli.core.commands import LongRunningOperation
+
+# pylint: disable=line-too-long
```

Comment:
how is debug difference from info? debug is at a less visible level than info right?

Reply:
Yes, but it can be helpful for logging information that isn't useful to users in general but helpful when they or we need to debug the command or their use of the command. You don't need to add much, but run a cli command with `--debug` and you will see that some extra information is logged.